common/app/experiments/Experiments.scala

```diff
@@ -22,10 +22,10 @@ object OldTLSSupportDeprecation extends Experiment(
   participationGroup = TLSSupport
 )
 
-object DotcomRenderingAdvertisements extends Experiment(
+object DotcomRendering extends Experiment(
   name = "dotcom-rendering-advertisements",
```

Comment:
Do you need to change this too or will we bosh all the exiting users out?

Reply:
Yes, in a different PR :)

Reply:
Didn't do it now because I will need to check the front end admin console.